<launch>
    <!-- パラメータ設定セクション nodeとは独立させて書く-->
    <param name="save_dir_name" value="/home/rokuto/"/>
    <param name="subscribe_topic_name" value="/livox/lidar"/>
    <param name="number_of_iterations" value="2"/>
    <param name="sample_rate" value="0.5"/>
    <param name="noise_variance" value="0.1"/>
    <param name="min_distance" value="2.5"/>
    <param name="max_distance" value="200"/>
    <param name="min_height" value="-0.5"/>
    <node name="rosbag_to_xyz" pkg="spoofing_sim" type="bag2test.py" output="screen" />
</launch>